Makefile

```diff
@@ -25,12 +25,13 @@ setversion:
 	@echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
 	@echo "Please set a tag like:  git tag 3.17"
 
+POS = $(wildcard po/*.po)
 translate:
 	grunt nggettext_extract
-	for language in de nl fr cs ; do \
-		(cd po; msgmerge $$language.po template.pot > tmp.po; mv tmp.po $$language.po) ; \
+	for language in $(POS); do \
+		(msgmerge $$language po/template.pot > tmp.po; mv tmp.po $$language) ; \
 	done
-	grunt nggettext_compile
+	grunt nggettext_compile --force
```

Comment:
why is there a `--force` needed?

Reply:
This is a good question, maybe some of the myriads translations is incomplete, I am getting this warning:

~~~~
Running "nggettext_compile:all" (nggettext_compile) task
Warning: No Language header found! Used --force, continuing.
~~~~

Without the `--force` the Makefile would break.

Reply:
OK, in my local copy I had some broken remnants like `de.po.po` from a previous Make run. I can remove the --force again.